README.md

```diff
@@ -1 +1,72 @@
-# Group-Capstone-
\ No newline at end of file
+# Screenshot
+[screenshot](src/groupCapstone.png) 
+
+[video presentation](https://drive.google.com/file/d/19XV32Kyiy6z-5QU6gO0r5RLOhZt1G9ZI/view?usp=sharing)
+
+# Group Capstone 
+
+In this project we worked collaboratively as a Group to implement a web app wich shows information about movie shows 
+
+The app fetches data from the tvmaze Api and users can interactively add likes to their favourite shows 
+
+The greatest feature is making reservations and making comments and moreso read other people comments about the movie show 
+
+If you like what you see and have a project you need coded, don't hesitate to contact us.
+
+
+# Built With
+
+
+- Major languages: HTML and CSS,
+- Technologies used: Web services, webpack, API
+
+
+# Getting Started
+
+
+To get a local copy up and running follow these simple example steps.
+
+## Prerequisites 
+- A working Laptop or Desktop
+## Setup
+- Install code editor
+- Install Git bash
+- Install webpack
+## Install
+- clone the [project](https://github.com/mutinhiri/Group-Capstone-).
+## Usage
+- Test your GitHub flow, a linter and professional documentation ability
+## Run tests
+- Run npm run build to rebuild the project
+- Run npm run start to run the project
+## Deployment
+- Push the project to your GitHub repo
+- Navigate to page in the project settings and publish
+# Authors
+
+  #  Bunbee
+  - GitHub: [Bunbee](https://github.com/mutinhiri).
+  - Twitter: [@Bunbee02](https://twitter.com/Bunbee02).
+  - LinkedIn: [@bunbee](https://www.linkedin.com/in/bunbee).
+
+# Amel
+  -Github: [Amel](https://github.com/)
+  -Twitter[@Amal Qahwadji](https://twitter.com/AmalQahwadji)
+  -LinkedIn[Amel Khiri](https://www.linkedin.com/in/amel-khiri-qahwadji-37a550135/)
+
+
+#  Anibal
+  -Github[Anibal Amoroso](https://github.com/sj1978)
+  -Twitter[@AmorosoAnibal](https://twitter.com/AmorosoAnibal)
```

Comment:
- [ ] Kindly make sure that the authors section has consistent formatting and that all links are working. (In `markdown` you should leave a `space` between the `-` and the first to letter to create a list item)